addons/stock/static/src/xml/forecast_widget.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <templates id="template" xml:space="preserve">
     <t t-name="stock.forecastWidget">
-        <span t-if="reservedAvailability" t-esc="reservedAvailability"/>
-        <span t-elif="!expectedDate" style="color: red;">Not Available</span>
-        <span t-else="" t-att-style="'color: ' + (isLate ? 'red;' : 'orange;')">Exp <t t-esc="expectedDate"/></span>
-        <button t-if="!hideReportButton" class="o_forecast_report_button btn btn-link o_icon_button ml-2" title="Forecasted Report">
+        <span t-if="['partially_available', 'assigned'].includes(state) or product_type != 'product'" t-esc="reserved_availability_str"/>
+        <span t-elif="!forecast_expected_date_str and forecast_availability" t-esc="reserved_availability_str"/> <!-- TODO: If have some stock -> print 0.00 even if not completly ??? -->
+        <span t-elif="forecast_expected_date_str" t-att-style="'color: ' + (forecast_is_late ? 'red;' : 'orange;')">Exp <t t-esc="forecast_expected_date_str"/></span>
+        <span t-else="" style="color: red;">Not Available</span>
+        <button t-if="product_type == 'product'" class="o_forecast_report_button btn btn-link o_icon_button ml-2" title="Forecasted Report">
```

Comment:
Same comment as [this one](https://github.com/odoo/odoo/pull/56425/commits/aba778e28789d7f3b563856c8ebdd6b2dff185c0#r484420280).